Update pycapnp to 0.5.12

..you knew it was coming :wink: 

Update pycapnp to the latest version.

For the record, I was getting a cryptic error when trying to install pycapnp: 

```
Collecting pycapnp==0.5.8
  Using cached pycapnp-0.5.8.tar.gz
Building wheels for collected packages: pycapnp
  Running setup.py bdist_wheel for pycapnp ... error
  Complete output from command /mnt/store/devel/HTM/pyenv/bin/python2 -u -c "import setuptools, tokenize;__file__='/tmp/pip-build-G0VstQ/pycapnp/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" bdist_wheel -d /tmp/tmpkdtrnVpip-wheel- --python-tag cp27:
  running bdist_wheel
  running build
  running build_py
  creating build
  creating build/lib.linux-x86_64-2.7
  creating build/lib.linux-x86_64-2.7/capnp
  copying capnp/_gen.py -> build/lib.linux-x86_64-2.7/capnp
  copying capnp/version.py -> build/lib.linux-x86_64-2.7/capnp
  copying capnp/__init__.py -> build/lib.linux-x86_64-2.7/capnp
  copying capnp/__init__.pxd -> build/lib.linux-x86_64-2.7/capnp
  copying capnp/schema.capnp -> build/lib.linux-x86_64-2.7/capnp
  copying capnp/c++.capnp -> build/lib.linux-x86_64-2.7/capnp
  creating build/lib.linux-x86_64-2.7/capnp/helpers
  copying capnp/helpers/helpers.pxd -> build/lib.linux-x86_64-2.7/capnp/helpers
  copying capnp/helpers/non_circular.pxd -> build/lib.linux-x86_64-2.7/capnp/helpers
  copying capnp/helpers/__init__.pxd -> build/lib.linux-x86_64-2.7/capnp/helpers
  copying capnp/helpers/serialize.h -> build/lib.linux-x86_64-2.7/capnp/helpers
  copying capnp/helpers/asyncHelper.h -> build/lib.linux-x86_64-2.7/capnp/helpers
  copying capnp/helpers/checkCompiler.h -> build/lib.linux-x86_64-2.7/capnp/helpers
  copying capnp/helpers/fixMaybe.h -> build/lib.linux-x86_64-2.7/capnp/helpers
  copying capnp/helpers/rpcHelper.h -> build/lib.linux-x86_64-2.7/capnp/helpers
  copying capnp/helpers/capabilityHelper.h -> build/lib.linux-x86_64-2.7/capnp/helpers
  creating build/lib.linux-x86_64-2.7/capnp/includes
  copying capnp/includes/schema_cpp.pxd -> build/lib.linux-x86_64-2.7/capnp/includes
  copying capnp/includes/types.pxd -> build/lib.linux-x86_64-2.7/capnp/includes
  copying capnp/includes/capnp_cpp.pxd -> build/lib.linux-x86_64-2.7/capnp/includes
  copying capnp/includes/__init__.pxd -> build/lib.linux-x86_64-2.7/capnp/includes
  creating build/lib.linux-x86_64-2.7/capnp/lib
  copying capnp/lib/capnp.pxd -> build/lib.linux-x86_64-2.7/capnp/lib
  copying capnp/lib/__init__.pxd -> build/lib.linux-x86_64-2.7/capnp/lib
  copying capnp/lib/pickle_helper.py -> build/lib.linux-x86_64-2.7/capnp/lib
  copying capnp/lib/__init__.py -> build/lib.linux-x86_64-2.7/capnp/lib
  copying capnp/lib/capnp.pyx -> build/lib.linux-x86_64-2.7/capnp/lib
  creating build/lib.linux-x86_64-2.7/capnp/templates
  copying capnp/templates/module.pyx -> build/lib.linux-x86_64-2.7/capnp/templates
  copying capnp/templates/setup.py.tmpl -> build/lib.linux-x86_64-2.7/capnp/templates
  running build_ext
  creating /tmp/tmpI_5Ac8/tmp
  cc -c /tmp/timer_createaN1qDv.c -o /tmp/tmpI_5Ac8/tmp/timer_createaN1qDv.o
  /tmp/timer_createaN1qDv.c:1:1: warning: return type defaults to ‘int’ [-Wimplicit-int]
   main (int argc, char **argv) {
   ^
  /tmp/timer_createaN1qDv.c: In function ‘main’:
  /tmp/timer_createaN1qDv.c:2:5: warning: implicit declaration of function ‘timer_create’ [-Wimplicit-function-declaration]
       timer_create();
       ^
  cc /tmp/tmpI_5Ac8/tmp/timer_createaN1qDv.o -o /tmp/tmpI_5Ac8/a.out
  /tmp/tmpI_5Ac8/tmp/timer_createaN1qDv.o: In function `main':
  timer_createaN1qDv.c:(.text+0x15): undefined reference to `timer_create'
  collect2: error: ld returned 1 exit status
  creating tmp
  creating tmp/tmpI_5Ac8
  cc -c /tmp/tmpI_5Ac8/vers.cpp -o tmp/tmpI_5Ac8/vers.o --std=c++11
  /tmp/tmpI_5Ac8/vers.cpp:4:26: fatal error: capnp/common.h: No such file or directory
  compilation terminated.
  *WARNING* no libcapnp detected. Will download and build it from source now. If you have C++ Cap'n Proto installed, it may be out of date or is not being detected. Downloading and building libcapnp may take a while.
  fetching https://capnproto.org/capnproto-c++-0.5.2.tar.gz into /tmp/pip-build-G0VstQ/pycapnp/bundled
  error: [Errno 13] Permission denied
  
  ----------------------------------------
  Failed building wheel for pycapnp
  Running setup.py clean for pycapnp
Failed to build pycapnp
Installing collected packages: pycapnp
  Running setup.py install for pycapnp ... error
    Complete output from command /mnt/store/devel/HTM/pyenv/bin/python2 -u -c "import setuptools, tokenize;__file__='/tmp/pip-build-G0VstQ/pycapnp/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-RuiNNZ-record/install-record.txt --single-version-externally-managed --compile --install-headers /mnt/store/devel/HTM/pyenv/include/site/python2.7/pycapnp:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-G0VstQ/pycapnp/setup.py", line 92, in <module>
        from Cython.Distutils import build_ext as build_ext_c
    ImportError: No module named Cython.Distutils

    ----------------------------------------
Command "/mnt/store/devel/HTM/pyenv/bin/python2 -u -c "import setuptools, tokenize;__file__='/tmp/pip-build-G0VstQ/pycapnp/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-RuiNNZ-record/install-record.txt --single-version-externally-managed --compile --install-headers /mnt/store/devel/HTM/pyenv/include/site/python2.7/pycapnp" failed with error code 1 in /tmp/pip-build-G0VstQ/pycapnp/
```

And it was caused by my temp dir (`/tmp/`) being mounted with `noexec`.

--- a/README.md
+++ b/README.md
@@ -37,7 +37,6 @@ Important notes:
 The Python dependencies (NumPy and pycapnp) can be installed with `pip`:
 
     pip install -r bindings/py/requirements.txt
-    pip install pycapnp==0.5.8
 
 ### Simple Source Installation (does not support incremental builds)
 

--- a/bindings/py/requirements.txt
+++ b/bindings/py/requirements.txt
@@ -3,4 +3,4 @@ pytest==2.5.1
 pytest-cov==1.6
 pytest-xdist==1.8
 numpy==1.11.2
-pycapnp==0.5.8
+pycapnp==0.5.12

